server/src/models/UserModel.js

```diff
@@ -3,10 +3,18 @@ const getResponseObject = require('../util/getResponseObject');
 
 const UserSchema = new mongoose.Schema(
   {
-    name: {
+    firstName: {
       type: String,
       required: true,
     },
+    lastName: {
+      type: String,
+      required: true,
```

Comment:
Try to follow this: https://mongoosejs.com/docs/validation.html to implement the validation you were doing in the validateUser function here